Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -44,10 +44,20 @@ def self.audit_args
       switch "--eval-all",
              description: "Evaluate all available formulae and casks, whether installed or not, to audit them. " \
                           "Implied if `HOMEBREW_EVAL_ALL` is set."
-      switch "--new", "--new-formula", "--new-cask",
+      switch "--new",
              description: "Run various additional style checks to determine if a new formula or cask is eligible " \
-                          "for Homebrew. This should be used when creating new formula and implies " \
+                          "for Homebrew. This should be used when creating new formulae or casks and implies " \
                           "`--strict` and `--online`."
+      switch "--new-formula",
+             description: "Run various additional style checks to determine if a new formula is eligible " \
+                          "for Homebrew. This should be used when creating new formulae and implies " \
+                          "`--strict` and `--online`.",
+             replacement: "--new"
+      switch "--new-cask",
+             description: "Run various additional style checks to determine if a new cask is eligible " \
+                          "for Homebrew. This should be used when creating new casks and implies " \
+                          "`--strict` and `--online`.",
+             replacement: "--new"
```

Comment:
```suggestion
             replacement: "--new",
             hidden: true 
```

so we can treat this as deprecated

Reply:
Currently, specifying `replacement` implies it is already `odeprecated` inside `parser.rb`. I guess we just make it `hidden` without `replacement`.